### PR TITLE
Store Orders: Update order total display after refunds

### LINKED ISSUE #19053: Store Orders: Should order total display the total after partial refunds?
If an order is partially refunded, we show the original total, then the -$X refunded below that, but we don't tell the user what the remaining total is -- what money they're actually still getting. Same thing on the order list page, we list order totals without indicating any refunded orders.

Example:

calypso

In wp-admin, the total amount is crossed out next to a new value, the remaining order total after the refund:

wp-admin

I discovered this because I'm (accidentally?) including the refund in my total calculations when editing orders in #19047. IMO we should add the total-minus-refund somewhere on this table, and also on the orders list table. (If we don't, I should update `getOrderTotal` to not include the refund)

cc @kellychoffman @jameskoster

### PR DESCRIPTION
**Pull request by ryelle**
Fixes #19053 — The order total was still displaying as full value even when the order has been partially or fully refunded. This PR updates the view with a "remaining total" field on orders with refunds, displaying the "real" current total (post-refund):

refunded

This also updates the list view with the full total crossed out, with the remaining total visible:

list-view

To test:

- View your orders list, see that any refunded orders have a crossed-out total + remaining total
- View an order with a refund (grant yourself a partial refund if needed)
- You should see the refund + remaining total on the single order view
- The refund view goes away when editing, because it's not an editable field
- Check orders without refunds, there should be no refund totals listed

### CODE REVIEW

File: client/extensions/woocommerce/app/order/order-details/table.js [hunk 1]
```diff
@@ -261,13 +261,46 @@ class OrderDetailsTable extends Component {
...
+		}
+		const showTax = this.shouldShowTax();
+		const totalValue = getCurrencyFormatDecimal( order.total, order.currency ) + refundValue;
+
+		return (
+			<Fragment>
+				<OrderTotalRow
+					className="order-details__total-refund"
+					currency={ order.currency }
+					label={ translate( 'Refunded' ) }
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/order/order-details/table.js]: Hi! I've found a possible matching string that has already been translated **38** times:
`translate( 'Refunded To' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/store-order-refund-display)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by kellychoffman** [comment]
Nice! 

> This also updates the list view with the full total crossed out, with the remaining total visible

screen shot 2017-12-15 at 10 14 01 am


The only issue I see here is its not immediately clear what he crossed out total is all about. Would it make sense to change the status from `Paid in full` to `Partially refunded`?

**Comment by ryelle** [comment]
>  It makes me think of a coupon/discount being applied. Maybe Kelly & Jay have some ideas.

Good point - we'll need to address that concept ASAP too, since it'll need to be clear in order creation how coupons affect the prices, see #16662

**Comment by kellychoffman** [comment]
"Partially refunded" status looks good 👍

**Pull request merged**